admin/data/sql/migrations/20200120144958.do.drop-pupilstatus_part_5.sql

```diff
@@ -0,0 +1,10 @@
+IF NOT EXISTS (SELECT * FROM sys.indexes i
```

Comment:
i believe we should just drop this anyway, as it relates to last years model/schema

Reply:
Dropped as requested